src/System.Private.ServiceModel/tools/test/SelfHostWcfService/CertificateResources/CertificateResourceHelpers.cs

```diff
@@ -37,10 +37,10 @@ internal static CertificateGenerator GetCertificateGeneratorInstance(BridgeConfi
                     {
```

Comment:
Ditto -- I don't think this class is used by the new infrastructure, so I don't think we should change it.  Ignore this comment if I'm wrong.